.github/workflows/action-build.yml

```
@@ -28,7 +28,7 @@ jobs:
 
       - uses: actions/setup-node@v3
```

wonder if this version could be bumped to:

```suggestion
        uses: actions/setup-node@v6
```

---

Same here opened a dedicated pr : https://github.com/release-drafter/release-drafter/pull/1468